src/codeu/chat/client/core/UserContext.java

```diff
@@ -34,6 +40,68 @@ public UserContext(User user, BasicView view, BasicController controller) {
     this.view = view;
     this.controller = controller;
   }
+  
+  public User addUserInterest(String name) {
+	  Uuid idToSave = null;
+	  // find the user with the same name as the one given 
+	  for(User u: view.getUsers()) {
+		  if (name.equals(u.name)){
+			  idToSave = u.id;
+		  }
+	  }
+	  if (idToSave == null)
+		  return null;
+	  // adds to the current user object stored on client
+	  user.addUserInterest(idToSave);
+	  // sends info to model to be saved also
+	  controller.newUserInterest(user.id, idToSave);
+	  return user;
+  }
+  
+  public User addConvoInterest(String title) {
+	  ConversationContext convoToSave = null;
+	  for(ConversationContext c: conversations()) {
+		  if (title.equals(c.conversation.title)){
+			  convoToSave = c;
+		  }	
+	  }
+	  if (convoToSave == null)
```

Comment:
nit: I'd always use `{}` curly brackets with `if` statements, even if they only contain one line.

Link to the "official" Java style guide: http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-142311.html

Link to Google's Java style guide: https://google.github.io/styleguide/javaguide.html#s4.1-braces